cookbooks/fb_helpers/recipes/spec.rb

```diff
@@ -20,6 +20,11 @@
 
 # This recipe is only for running ChefSpec tests
 if defined?(ChefSpec)
+  # network scripts uses the fb_modprobe_module resource which depends
+  # on the execute[load modules] resource in systemd
+  include_recipe 'fb_systemd'
+  include_recipe 'fb_modprobe'
+  include_recipe 'fb_network_scripts'
```

Comment:
Pulling in a bunch of largely unrelated recipes (and their dependencies) will make this horribly brittle, and make the deps explode for fb_helpers.

Backing up - what's actually breaking here?  I can't infer what systemd has to do with this spec.  And I'm failing to find the original error which is showing up in GH

Reply:
Well the fb_systemd is for:
```
       expected Chef::Exceptions::UserIDNotFound, got #<NoMethodError: undefined method `fb_systemd_reload' for cookbook: fb_systemd, recipe: reload :Chef::Recipe> with backtrace:
```

Then, per the comments, once you load fb_systemd, then it want modprobe.

Not sure I remember what network_scripts was for, but my guess is another error popped up after that.

Reply:
Looks like whatever the dep was is gone. will close.